### DISCUSSION TITLE
Moving a bone node to a specific location

### DISCUSSION THREAD

**Discussion by branaway** [pixiv/three-vrm#1235] [Help]
For an avatar system like VRM, is there a simple way of moving a bone node to a specific position relative to the body center? e.g., can I simply issue a command to move the right hand to the forehead in a solute position, or to the the front of the chest with the palm facing inward? Any ideas for a workaround if there is no such utility?

**Comment by ke456-png** [comment]
Does that mean Inverse Kinematics?

If you could provide us with specific features that you would like to achieve, it would be very helpful.
